src/libcore/task/wake.rs

```diff
@@ -131,9 +131,9 @@ impl RawWakerVTable {
     /// resources that are associated with this instance of a [`RawWaker`] and
     /// associated task.
     #[rustc_promotable]
-    #[unstable(feature = "futures_api",
-            reason = "futures in libcore are unstable",
-            issue = "50547")]
+    #[cfg_attr(stage0, unstable(feature = "futures_api_const_fn_ptr", issue = "50547"))]
+    #[cfg_attr(not(stage0), stable(feature = "futures_api", since = "1.35.0"))]
+    #[cfg_attr(not(stage0), rustc_allow_const_fn_ptr)]
```

Comment:
+ Please add a comment making it clear that no other function should use this attribute and that T-Lang should be contacted for any new wanted uses.

+ Also add a FIXME for removing the hack whenever we have a stable way to accept function pointers, cc https://github.com/rust-lang/rfcs/pull/2632.